Makefile

```diff
@@ -41,27 +48,44 @@ db-seed-all:
 # ===============================================
 
 # Run the devserver, and live reload our CSS and JS.
-devserver:
+devserver: py-venv-check
 	npx concurrently "flask run" "make css-dev" "make js-dev"
 
 # Start using Docker.
 start-docker:
 	docker-compose up --build --force-recreate
 
 # Run a local Celery instance for background tasks.
-celery:
+celery: 
 	celery -A ambuda.tasks worker --loglevel=INFO
 
+lint-isort:
+	@echo "$(BOLD) Running isort to organize python imports"
```

Comment:
didn't know about this `@` syntax -- very cool!

Where/how is `BOLD` defined? I don't seem to have support for it.

Reply:
`@` is comes handy for pretty printing. Suppresses Makefile's line command and prints the executed commands output.
https://www.gnu.org/software/make/manual/make.html#Recipe-Echoing

I removed BOLD. Not needed at this time.